restcomm/restcomm.dao/src/main/resources/org/restcomm/connect/dao/mybatis/sms-messages.xml

```diff
@@ -117,8 +117,12 @@
                 AND "date_created" &gt;= #{startTime}
             </if>
             <if test="endTime != null">
-                    AND "date_created" &lt;= DATE_ADD(#{endTime},INTERVAL 1 DAY)
+                AND "date_created" &lt;= DATE_ADD(#{endTime},INTERVAL 1 DAY)
+            </if>
+            <if test="smppMessageId != null">
```

Comment:
we need to make this change for `sms-messages.xml` files in `restcomm.application` module as they are the ones that are packaged and used on production

Reply:
good point, agree